code_folder/C0_data_processing.py

```diff
@@ -106,7 +128,7 @@ def process_all_assets(dict_values):
         # Add lifetime capex (incl. replacement costs), calculate annuity (incl. om), and simulation annuity to each asset
         for asset in dict_values["energyConversion"]:
             helpers.define_missing_cost_data(
-                dict_values["economic_data"], dict_values["energyConversion"][asset]
+                dict_values, dict_values["energyConversion"][asset]
```

Comment:
Why did you have to change "economic data" here?

Reply:
because I needed the simulation_settings as well. So it was easier to send the whole dictionary